managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,433 @@
+
+- **Username**: The username to use to connect to Red Hat Virtualization.
+
+- **Password**: The password to use to connect to Red Hat Virtualization.
+
+- **Host (Authentication URL)**: The host to be used for authentication.
+
+   **Important**: Enter in **Host** the Red Hat Virtualization provider URL, followed by the path `/ovirt_engine/api`. Example: `https://your.rhv.com/ovirt_engine/api`
+
+##### VMware
+
+Selecting this credential type enables synchronization of inventory with VMware vCenter.
+
+**Important:** If both {{ site.data.product.title_short }} and a VMware provider are located in the same IPv6-only network, use a DNS-resolvable hostname for the VMware provider in the **vCenter Host** field when adding credentials.
+
+VMware credentials have several attributes that can be configured:
+
+- **Username**: The username to use to connect to vCenter.
+
+- **Password**: The password to use to connect to vCenter.
+
+- **vCenter Host**: The vCenter hostname or IP address to connect to.
+
+**Note:** If the VMware guest tools are not running on the instance, VMware inventory sync might not return an IP address for that instance.
+
+### Authoring a workflow
+
+When the *Embedded Workflows* feature is enabled, you can configure workflows for use in the following areas during service catalog creation:
+
+- **Dynamic dialog fields** - You (or your users) can attach workflows to a field so that the field becomes populated or refreshed with the results from the running of the workflow.
+- **Service Entry Points** - You (or your users) can set and use workflows for provisioning, reconfiguring, and retiring services.
+
+The Automate model provides flexibility to not only change parts of the provisioning process, but also to allow you to automate other operational tasks. You can create and use your own embedded workflows as needed to automate your operational tasks.
+
+Workflows must be authored in Amazon State Languages (ASL) format. As part of authoring a workflow, you (or your users) must build and support the containers that are run during each step of an embedded workflow.
+
+1. Define the code for the workflow. If your workflow requires the use of any credentials or parameters to be specified, ensure that they are passed in the code.
+
+   The workflow code must be in the Amazon States Language (ASL) format and follow its supported specifications. For more information about Amazon States Language and its specification, see [Amazon States Language Guide](https://states-language.net/).
+
+   Within the workflow code, you need to specify the states that your workflow requires, including any next steps. For each of the steps in the workflow, a docker container is called. The container defines what happens for that state. For example, a docker container can run to clone a template.
```

Comment:
> For each of the steps in the workflow, a docker container is called.

A Docker container is only run for `Task` type states, not every state.

Reply:
I have updated this text from:
> Within the workflow code, you need to specify the states that your workflow requires, including any next steps. For each of the steps in the workflow, a docker container is called. The container defines what happens for that state.

to be: 

> Within the workflow code, you need to specify the states that your workflow requires, including any next steps. For `Task` type steps in the workflow, a docker container is called. The container defines what happens for that Task state.